Resources/views/Partials/_menu.html.twig

```diff
@@ -8,11 +8,28 @@
     {% if item.labelAttribute('data-image') %}<img src="{{ item.labelAttribute('data-image') }}" alt="{{ item.name }}" class="menu-thumbnail"/>{% endif %}
     {% if item.hasChildren and options.depth is not same as(0) and item.displayChildren %}
         <span class="pull-right-container">
+            {{ _self.badges(item) }}
```

Comment:
The `_self` is not available for every compatible twig version: 
https://github.com/kevinpapst/AdminLTEBundle/blob/master/composer.json#L29
https://twig.symfony.com/doc/2.x/tags/macro.html
So this could be a BC break - please import it (eg [like here](https://github.com/kevinpapst/AdminLTEBundle/blob/master/Resources/views/Macros/default.html.twig#L27))

Reply:
I used `import _self as selfMacros` to prevent BC break